infrastructure/tools/Dockerfile

```diff
@@ -11,5 +11,6 @@ RUN apt-get update
 RUN apt-get install -y git fabric liblua5.2 liblua5.2-dev python3-dev python3-pip
 RUN pip3 install --upgrade pip
 RUN pip3 install wheel
+RUN pip3 install websockets
```

Comment:
@oherasym, why you put module install here? Whell ia installed outside because without it requrements.txt wil not work. But the rest packs should be installed through  requirements